addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -211,22 +208,20 @@ private void updateFanartUrl(String imagePath) {
         /*
```

Comment:
Maybe we can skip this here. Tomorrow we got a contribution (PR #2304) which will fix it.